pkg/scheduler/algorithm/priorities/resource_allocation.go

```diff
@@ -99,5 +99,14 @@ func getNonZeroRequests(pod *v1.Pod) *schedulernodeinfo.Resource {
 		result.MilliCPU += cpu
 		result.Memory += memory
 	}
+
+	// If Overhead is being utilized, add to the total requests for the pod
+	if pod.Spec.Overhead != nil && utilfeature.DefaultFeatureGate.Enabled(features.PodOverhead) {
+		cpu, memory := priorityutil.GetNonzeroRequests(&pod.Spec.Overhead)
```

Comment:
This means if cpu/memory are not set in pod.Spec.Overhead, a default value of .1 vcpu/200MB will be returned. Is it the intention that those are the minimum overheads?

Reply:
Got it - thanks @ahg-g - this is definitely not the intention.  TBH, how  nonzero requests are used (and how they are related to the total requests) was not very clear to me when following the code/definitions.

It *sounds* like GetNonZero requests wouldn't need Overhead then?  How's `non0CPUReq, non0MemReq` used?

The intent is to take into account the overhead associated with running a sandbox (pod), and from an "allocatable" standpoint to take this into account.

Reply:
As we discussed over slack, we should have a separate function that basically return 0 memory and cpu if their values were not set.

Reply:
Thanks for the help @ahg-g.  updated.